zeebe/broker/src/test/java/io/camunda/zeebe/broker/exporter/metrics/MetricsExporterTest.java

```diff
@@ -66,6 +82,13 @@ void shouldObserveJobLifetime() {
         .map(s -> s.value)
         .describedAs("Expected exactly 1 observed job_life_time sample counted")
         .containsExactly(1d);
+    assertThat(metrics.getJobLifeTime().collect())
```

Comment:
:thought_balloon: This isn't an ideal validation, but it does find the error. Without the fix, here you only get the `[0, Inf+]` label values (0 being the partition ID, `Inf+` being the `le` label).